recipe/meta.yaml

```diff
@@ -1,27 +1,27 @@
 {% set name = "aws-c-cal" %}
-{% set version = "0.6.9" %}
+{% set version = "0.5.20" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/awslabs/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 46defd807034ca9d873cc1459bc60f0bb4662029f7120db1af464421907fb480
+  sha256: acc352359bd06f8597415c366cf4ec4f00d0b0da92d637039a73323dd55b6cd0
 
 build:
-  number: 3
+  number: 0
   run_exports:
     - {{ pin_subpackage("aws-c-cal", max_pin="x.x.x") }}
```

Comment:
What's the purpose of this line?

Reply:
exports symbol implicitly in child package by including it only in host section, you can omit the run section then

Reply:
@Jrice1317 see https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#export-runtime-requirements.

In short, it will instruct `conda-build` to inject `aws-c-cal` `x.x.x` into a package's runtime requirements if `aws-c-cal` is found in the `host` section.

For example, let's say we have this hypothetical recipe:
```yaml
package:
  name: something
  version: 1.0.0

requirements:
  host:
    - aws-c-cal 0.5.20
```
the resulting `something` package would have `aws-c-cal=0.5.20` in its runtime dependencies because conda-build would have injected that due to the `run_exports` in `aws-c-cal`.

Reply:
Oh wow! Thanks for sharing that link and example! That makes a ton of sense!